ui/src/views/ColumnMappingEdit.vue

```diff
@@ -1,36 +1,121 @@
 <template>
-  <side-pane :is-open="true" title="Edit column mapping" @close="onCancel" v-if="columnMapping">
-    <p><strong>TODO</strong></p>
-    <div class="content">
-      <ul>
-        <li>{{ columnMapping.sourceColumn.id.value }}</li>
-        <li>{{ columnMapping.targetProperty.value }}</li>
-        <li>{{ columnMapping.datatype }}</li>
-        <li>{{ columnMapping.language }}</li>
-      </ul>
-    </div>
+  <side-pane :is-open="true" :title="operation.title" @close="onCancel">
+    <hydra-operation-form
+      :operation="operation"
+      :resource="resource"
+      :shape="shape"
+      :error="error"
+      :is-submitting="isSubmitting"
+      @submit="onSubmit"
+      @cancel="onCancel"
+    />
   </side-pane>
 </template>
 
 <script lang="ts">
 import { Vue, Component } from 'vue-property-decorator'
 import { namespace } from 'vuex-class'
-import { ColumnMapping } from '@cube-creator/model'
+import { RuntimeOperation } from 'alcaeus'
+import { ColumnMapping, Table, CsvSource } from '@cube-creator/model'
+import clownface, { GraphPointer } from 'clownface'
+import { Shape } from '@rdfine/shacl'
+import { APIErrorValidation, ErrorDetails } from '@/api/errors'
 import SidePane from '@/components/SidePane.vue'
+import HydraOperationForm from '@/components/HydraOperationForm.vue'
+import { rdf, schema } from '@tpluscode/rdf-ns-builders'
+import { dataset } from '@rdf-esm/dataset'
+import { api } from '@/api'
 
 const projectNS = namespace('project')
 
 @Component({
-  components: { SidePane },
+  components: { SidePane, HydraOperationForm },
 })
 export default class CubeProjectEditView extends Vue {
   @projectNS.Getter('findColumnMapping') findColumnMapping!: (id: string) => ColumnMapping | null
+  @projectNS.Getter('findTable') findTable!: (id: string) => Table
+  @projectNS.Getter('findSource') findSource!: (id: string) => CsvSource
+  @projectNS.Getter('sources') sources!: CsvSource[]
+
+  resource: GraphPointer | null = null
+  isSubmitting = false;
+  error: ErrorDetails | null = null;
+  shape: Shape | null = null;
 
   get columnMapping (): ColumnMapping | null {
     const columnMappingId = this.$router.currentRoute.params.columnMappingId
     return this.findColumnMapping(columnMappingId)
   }
 
+  get table (): Table {
+    const tableId = this.$router.currentRoute.params.tableId
+    return this.findTable(tableId)
+  }
+
+  get source (): CsvSource | null {
+    for (const source of this.sources) {
+      if (source.columns.find((column) => column.id.value === this.columnMapping?.sourceColumn.id.value)) {
+        return source
+      }
+    }
```

Comment:
I am open for suggestions here

Reply:
It's fine. I'm reworking this form at the moment, so I'll probably do some refactoring anyway.